package/src/components/Message/MessageSimple/MessageSimple.tsx

```diff
@@ -234,16 +262,49 @@ const MessageSimpleWithContext = <
             {message.pinned ? <MessagePinnedHeader /> : null}
           </View>
 
-          <View style={[styles.contentWrapper, contentWrapper]}>
-            <MessageContent
-              backgroundColor={backgroundColor}
-              noBorder={noBorder}
-              setMessageContentWidth={setMessageContentWidth}
-            />
-            {reactionListPosition === 'top' && ReactionListTop ? (
-              <ReactionListTop messageContentWidth={messageContentWidth} />
-            ) : null}
-          </View>
+          {enableSwipeToReply ? (
+            alignment === 'left' ? (
+              <Swipeable
```

Comment:
Also, since these `Swipeable`s mostly have the same props you can probably do something like:

```
<Swipeable
                containerStyle={[styles.contentWrapper, contentWrapper]}
                friction={2}
                onSwipeableWillOpen={() => {
                  if (!swipeableRef.current) return;
                  clearQuotedMessageState();
                  setQuotedMessageState(message);
                  triggerHaptic('impactLight');
                  swipeableRef.current.close();
                }}
                ref={swipeableRef}
                {...(alignment === 'left' ? leftAlignmentProps : rightAlignmentProps)}
              >
                {renderMessageBubble}
              </Swipeable>
```

Reply:
And get rid of the duplicate declarations altogether

Reply:
Good point. Let me do this as well..

Reply:
Done